src/mscorlib/shared/System/ArgumentOutOfRangeException.cs

```diff
@@ -61,12 +63,13 @@ public ArgumentOutOfRangeException(String paramName, Object actualValue, String
         protected ArgumentOutOfRangeException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
-            throw new PlatformNotSupportedException();
+            _actualValue = info.GetValue("ActualValue", typeof(object));
         }
 
         public override void GetObjectData(SerializationInfo info, StreamingContext context)
         {
             base.GetObjectData(info, context);
```

Comment:
Same comment about renaming the field for safety? Also same comment about checking for null info. I'll stop commenting on these.